src/benchmark.php

```diff
@@ -35,6 +35,16 @@
     $content = file_get_contents($filename);
     $newContent = str_replace('private const', 'const', $content);
     file_put_contents($filename, $newContent);
+
+    $filenameArangoDb = '../vendor/triagens/arangodb/lib/ArangoDBClient/ConnectionOptions.php';
+    $contentArangoDb = file_get_contents($filenameArangoDb);
+    $newContentArangoDb = str_replace('DefaultValues::DEFAULT_CIPHERS', 'null', $contentArangoDb);
```

Comment:
I think that's not needed, you have to include that class `DefaultValues` manually before a thread is started. A newly started thread has no access to previous declared autoloader.

Reply:
Will check